ops/testing.py

```diff
@@ -552,6 +551,10 @@ def add_storage(self, storage_name: str, count: int = 1,
         location set to /[tmpdir]/<storage_name>N, where N is the counter and
         will be a number from [0,total_num_disks-1].
 
+        The test harness uses symbolic links to imitate storage mounts, which may lead to some
+        inconsistencies compared to the actual charm. Users should be cognizant of
+        this potential discrepancy.
```

Comment:
IMO we should drop the sentence "Users should be cognizant of..." (here and below). We include the first sentence exactly so users are cognizant of it.

Reply:
Removed, thanks!